.github/workflows/testing.yml

```diff
@@ -92,6 +92,8 @@ jobs:
 
   python-nightly:
     runs-on: ubuntu-18.04
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
Why is this needed?

Reply:
This is because of:

https://github.com/cloudpipe/cloudpickle/blob/master/.github/workflows/testing.yml#L103

Shouldn't we use the non-deprecated way to change the PATH instead?

https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files

Reply:
See: #413.